library/src/G4CMPKaplanQP.cc

```diff
@@ -75,7 +75,8 @@ G4double G4CMP::KaplanPhononQP(G4double energy,
 G4CMPKaplanQP::G4CMPKaplanQP(G4MaterialPropertiesTable* prop, G4int vb)
   : verboseLevel(vb), filmProperties(0), filmThickness(0.), gapEnergy(0.),
     lowQPLimit(3.), subgapAbsorption(0.), absorberGap(0.),
-    phononLifetime(0.), phononLifetimeSlope(0.), vSound(0.), temperature(0.) {
+    phononLifetime(0.), phononLifetimeSlope(0.), vSound(0.), temperature(0.), 
+    absorberEff(0.), absorberEffSlope(0.) {
```

Comment:
Make sure you declare these data members in the "private:" section of the class definition file (G4CMPKaplanQP.hh).  I think I would move them up a couple of lines, and declare (and initialize) them right after "absorberGap".